src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -27,6 +27,8 @@
     private final Optional<String> normalSample;
     final OverlapDetector<MinorAlleleFractionRecord> tumorSegments;
     public static final String FILTERING_STATUS_VCF_KEY = "filtering_status";
+    private final int ALT_FWD_INDEX = 2;
```

Comment:
Rather than explicitly declare array indices here it would be better to have a method like `public static int getAltForwardCountsFromFlattenedContingencyTable(final int[] contingencyTable)` inside `StrandBiasBySample`.